menuinst/data/menuinst-1.schema.json

```
@@ -1,30 +1,36 @@
 {
   "title": "MenuInstSchema",
-  "description": "Metadata required to create menu items across operating systems with ``menuinst``.",
+  "description": "Metadata required to create menu items across operating systems with `menuinst`.",
   "type": "object",
   "properties": {
     "$id": {
       "title": "$Id",
       "description": "DEPRECATED. Use ``$schema``.",
       "default": "https://schemas.conda.org/menuinst-1.schema.json",
       "deprecated": true,
+      "markdownDescription": "DEPRECATED. Use ``$schema``.",
```

```suggestion
      "markdownDescription": "DEPRECATED. Use `$schema`.",
```

Should it be single back tics like in the other instances?